File: Coding-Curriculum/assets/js/drawing.js
function drawLine(x1,y1,x2,y2)
{
    var c = document.getElementById("drawing");
    var ctx = c.getContext("2d");
    ctx.moveTo(x1,y1);
    ctx.lineTo(x2,y2);
    ctx.stroke();
}

function fillRect(x,y,width,height)
{
    var c = document.getElementById("drawing");
    var ctx = c.getContext("2d");
    ctx.fillRect(x,y,width,height);
}

function clearCanvas()
{
    var c = document.getElementById("drawing");
    var ctx = c.getContext("2d");
    ctx.clearRect(0, 0, 400, 400);
    ctx.fillStyle = "#FAEBD7";
    ctx.fillRect(0,0,400,400);
    ctx.fillStyle = "#000000";
}

File: Coursework/Coding-Curriculum/assets/ruby/drawing.rb
require 'js'

def drawLine(x1,y1,x2,y2)
  %x{drawLine(#{x1},#{y1},#{x2},#{y2});}
  end

def fillRect(x,y,width,height)
  %x{fillRect(#{x},#{y},#{width},#{height});}
end

def clearCanvas
  %x{clearCanvas();}
end
